synchronizer/synchronizer.go

```diff
@@ -1251,8 +1251,22 @@ func (s *ClientSynchronizer) processTrustedBatch(trustedBatch *types.Batch, dbTx
 
 		// Find txs to be processed and included in the trusted state
 		if *s.trustedState.lastStateRoot == batches[1].StateRoot {
+			// Delete txs that were stored before restart. We need to reprocess all txs because the intermediary stateRoot is only store in memory
```

Comment:
```suggestion
			// Delete txs that were stored before restart. We need to reprocess all txs because the intermediary stateRoot is only stored in memory
```